src/data/nutritions.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@alexfauquette labels should be translatable.

Reply:
I assume `{{ nutritiveValue.name }}` should become ``{{ $t(`nutrition.names.${nutritiveValue.name}`) }}``

Reply:
I add it to the todo list